users/schema.py

```diff
@@ -8,16 +10,24 @@
 
 from common.schema import LanguageEnum
 from common.utils import update_object
-from kukkuu.exceptions import ObjectDoesNotExistError
+from kukkuu.exceptions import InvalidEmailFormatError, ObjectDoesNotExistError
 
 from .models import Guardian
 
 User = get_user_model()
 
 
+def validate_guardian_data(guardian_data):
+    if "email" in guardian_data:
+        try:
+            EmailValidator()(guardian_data["email"])
```

Comment:
There is a `validate_email(<email>)` function from the same package which basically do the same thing but I think it looks nicer :D

Reply:
That is indeed way nicer 👍